packages/ui-extensions/src/shared.ts

```diff
@@ -24,6 +24,7 @@ export type ApiVersion =
   | '2024-07'
   | '2024-10'
   | '2025-01'
+  | '2025-04'
```

Comment:
I'm not sure if we want to add this yet. It's not a valid version yet so could be a little misleading. Usually we add this days before publishing.